site/pages/react/ui-components.mdx

```diff
@@ -83,42 +97,20 @@ The easiest way to configure your UI components, is using the [Demo App](https:/
 - `sections` object is an array of arrays representing the various sections in descending order, each section (element in the outer array) separated by a divider in the UI
 - each item within a section is an `AuthTypes` objects containing configuration parameters for that authentication method.
 
-  <br />
-
 **The following [`AuthTypes`](https://github.com/alchemyplatform/aa-sdk/blob/main/account-kit/react/src/components/auth/types.ts)
 can be passed into sections:**
 
 ### Email
 
-This adds an email login to a section. The shape of this auth type is:
-
-```ts
-type EmailAuthType = {
```

Comment:
Not sure if we wanted to keep types here.  I removed it because i was trying to reduce duplication since this now lives in the login method page for each of the auth types